package.json

```diff
@@ -19,103 +19,113 @@
     "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'bower_components', 'junction') } catch (e) { }\""
   },
   "devDependencies": {
+    "@babel/core": "^7.4.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.4",
+    "@babel/plugin-transform-block-scoping": "^7.4.4",
     "@ember/jquery": "^0.6.0",
-    "async": "^2.6.1",
+    "@ember/optional-features": "^0.7.0",
+    "@types/ember": "^3.1.0",
+    "@types/ember-data": "^3.1.7",
+    "@types/rsvp": "^4.0.2",
+    "async": "^2.6.2",
     "babel-eslint": "^8.2.3",
     "broccoli-asset-rev": "^3.0.0",
     "broccoli-persistent-filter": "^2.0.0",
     "ember-ajax": "5.0.0",
-    "ember-browserify": "^1.1.13",
-    "ember-cached-shoe": "^0.2.0",
-    "ember-cli": "~3.10.0",
-    "ember-cli-app-version": "^3.0.0",
+    "ember-auto-import": "^1.3.0",
+    "ember-cached-shoe": "^0.2.1",
+    "ember-cli": "~3.9.0",
+    "ember-cli-app-version": "^3.2.0",
     "ember-cli-autoprefixer": "0.8.1",
-    "ember-cli-babel": "^7.1.3",
-    "ember-cli-clipboard": "^0.11.0",
+    "ember-cli-babel": "^7.7.3",
+    "ember-cli-cjs-transform": "^1.3.1",
+    "ember-cli-clipboard": "^0.11.1",
     "ember-cli-code-coverage": "^1.0.0-beta.0",
-    "ember-cli-content-security-policy": "^1.0.0",
-    "ember-cli-dependency-checker": "^3.0.0",
+    "ember-cli-content-security-policy": "^1.1.1",
+    "ember-cli-dependency-checker": "^3.1.0",
     "ember-cli-deploy": "^1.0.0",
     "ember-cli-deploy-build": "^1.0.0",
     "ember-cli-deploy-git": "^1.2.0",
     "ember-cli-deploy-html-manifest": "0.0.6",
     "ember-cli-deploy-revision-data": "^1.0.0",
-    "ember-cli-deprecation-workflow": "^1.0.0",
-    "ember-cli-document-title": "^0.4.0",
-    "ember-cli-dotenv": "^2.0.0",
+    "ember-cli-deprecation-workflow": "^1.0.1",
+    "ember-cli-document-title-northm": "^1.0.3",
```

Comment:
@niranjan94 I replaced this because `ember-cli-document-title` is no longer maintained. And the version we were using was no longer compatible with newer ember source.
https://github.com/kimroen/ember-cli-document-title/issues/83#issuecomment-456837023

Reply:
Awesome. Did not notice that.. Will change